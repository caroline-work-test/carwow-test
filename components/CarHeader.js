import React, {Component} from 'react';

class CarHeader extends Component {
render () {
    return (
      <div>
      <h1 className='CarHeader_Title'>Car Wow Test</h1>
        <p className='CarHeader_Description'>
            {this.props.lead}
        </p>
      </div>)
  }
}

export default CarHeader;