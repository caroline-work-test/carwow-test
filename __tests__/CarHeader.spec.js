import * as React from 'react'
import {shallow} from 'enzyme'
import CarHeader from '../components/CarHeader';
import {createSerializer} from 'enzyme-to-json';
expect.addSnapshotSerializer(createSerializer({mode: 'deep'}));
import toJson from 'enzyme-to-json';

describe('CarHeader', () => {
  it('should render correctly', function () {
      const wrap = shallow(<CarHeader lead={'This is test lead.'}/>)
      expect(toJson(wrap)).toMatchSnapshot();
  })
  it('should contain prop', function () {
    const wrap = shallow(<CarHeader lead={'This is test lead.'}/>)
    expect(wrap.find('.CarHeader_Description').text()).toBe('This is test lead.')
})
})