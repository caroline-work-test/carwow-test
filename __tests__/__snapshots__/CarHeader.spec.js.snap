// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CarHeader should render correctly 1`] = `
<div>
  <h1
    className="CarHeader_Title"
  >
    Car Wow Test
  </h1>
  <p
    className="CarHeader_Description"
  >
    This is test lead.
  </p>
</div>
`;
